<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1c1d22;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>46</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>46</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>46</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #121116;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="header_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>46</y>
      <width>800</width>
      <height>3</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>3</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>3</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3f5ca7;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="entry">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>250</y>
      <width>550</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #121116;
border-radius: 3px;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>25</y>
       <width>504</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 0px;</string>
     </property>
     <property name="placeholderText">
      <string>Enter a movie name</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="attributes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="attSelection">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>130</y>
       <width>350</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-radius: 3px;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>320</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 0px;
border-left: 0px;</string>
      </property>
      <property name="text">
       <string>      Sort by recency</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>320</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 0px;
border-bottom: 0px;
border-left: 0px;</string>
      </property>
      <property name="text">
       <string>      Sort by normalized score</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>320</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 0px;
border-left: 0px;</string>
      </property>
      <property name="text">
       <string>      Sort by reviewer name</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QPushButton" name="ascButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>470</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>Sort ascending</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="descButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>470</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>Sort descending</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="results">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="resultsView">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>170</y>
       <width>650</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-radius: 3px;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="sortType">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>1</y>
        <width>626</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 0px;
border-right: 0px;
border-left: 0px;</string>
      </property>
      <property name="text">
       <string>Sorted by normalized score (descending)</string>
      </property>
     </widget>
     <widget class="QListWidget" name="resList">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>31</y>
        <width>626</width>
        <height>268</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchAgainButton">
     <property name="geometry">
      <rect>
       <x>585</x>
       <y>120</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>Search again</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="viewListButton">
     <property name="geometry">
      <rect>
       <x>585</x>
       <y>70</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>View my list</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>39</y>
       <width>450</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>8</y>
        <width>400</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Example Movie Title Here</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>8</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>♡</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="stats">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>42</y>
        <width>401</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Total Score: 5,535 | Mean Score: 52.8</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="stats_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>401</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>QS: ms | MS: ms</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="favorites">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="favView">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>130</y>
       <width>350</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-radius: 3px;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QListWidget" name="favList">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>348</width>
        <height>298</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>470</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #121116;
border-color: #7e7e7e;
border-width: 1px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>Search again</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>header</zorder>
   <zorder>header_2</zorder>
   <zorder>attributes</zorder>
   <zorder>entry</zorder>
   <zorder>results</zorder>
   <zorder>favorites</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
